deposit_contract/contracts/validator_registration.v.py

```diff
@@ -81,12 +85,11 @@ def deposit(deposit_input: bytes[512]):
 
     self.deposit_count += 1
     new_deposit_root: bytes32 = self.get_deposit_root()
-    log.Deposit(new_deposit_root, deposit_data, merkle_tree_index, self.branch)
+    log.Deposit(new_deposit_root, deposit_data, self.to_little_endian_64(index), self.branch)
```

Comment:
```suggestion
    log.Deposit(self.get_deposit_root(), deposit_data, self.to_little_endian_64(index), self.branch)
```

Reply:
Hey , so the reason I didnt use a function call here is because, it would be expensive in terms of gas to derive the merkle root multiple times. So I just derived it once and used that when emitting the deposit and chainstart logs instead.